lib/cli.js

```diff
@@ -235,12 +212,9 @@ const cli = {
 		} catch (error) {
 			debug("Error parsing CLI options:", error.message);
 
-			let errorMessage = error.message;
-
-			if (usingFlatConfig) {
-				errorMessage +=
-					"\nYou're using eslint.config.js, some command line flags are no longer available. Please see https://eslint.org/docs/latest/use/command-line-interface for details.";
-			}
+			const errorMessage = `${
+				error.message
+			}\nYou're using eslint.config.js, some command line flags are no longer available. Please see https://eslint.org/docs/latest/use/command-line-interface for details.`;
```

Comment:
I'm not sure if this message makes sense now because the CLI documentation doesn't show removed flags and how to replace them. Maybe this should point to https://eslint.org/docs/latest/use/configure/migration-guide#cli-flag-changes, although it's currently out-of-date.